docs/guides/quick-answers/linux/how-many-kb-in-a-mb/index.md

```diff
@@ -0,0 +1,81 @@
+keywords: ['how many kb in mb','mb','kb to gb','megabytes in a gigabyte']
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+published: 2022-03-31
+modified_by:
+  name: Linode
+title: "How Many Kilobytes are in a Megabyte?"
+h1_title: "Data Storage Guide: How Many KB are in a MB?"
+enable_h1: true
+contributor:
+  name: Jack Wallen
+---
+
+Kilobytes and Megabytes are units of measure that are used in nearly every computing platform on the planet. This goes for mobile phones, desktops, servers, virtual machines, containers, and cloud hosting. Being well-versed in these measurements is important in a variety of applications, but is especially important when you work with third-party cloud vendors that charge for bandwidth and storage . At a bare minimum, you need to calculate the average usage so you know how much that cloud service is going to cost.
+
+No matter the reason, when working as a system administrator or in any computer-related field, you need to know how many kilobits are in a megabyte and how many megabytes are in a gigabyte. After all, it's sometimes easy to mistakenly label something as a bit when you mean byte, or vice versa. This is also important to understand when you're promised a certain level of bandwidth, only to find out there's been confusion between bits and bytes.
+
+## What Is the Difference Between a Bit and a Byte?
+
+Both are a means to measure units of data. The smallest unit of measurement for data is the bit. A bit is a binary measurement, so it can either be 0 or 1. Every computer sends and receives data as a collection of ones and zeros (binary). Each 0 or 1 is a bit.
+
+Data that is transferred over a network connection is generally measured in bits with the following designations:
+
+| Name | Number of Bits |
+|:-------:|:-------:|
+| bit | 1 |
+ kilobits | 1,000 |
+| megabits | 1,000,000 |
+| gigabits | 1,000,000,000 |
+
+In the early days of the Internet, you had a modem that was either 14.4, 28.8, or 33.6. Those numbers represented how many bits per second the hardware could send or receive. So 14.4, 28.8, and 33.6 were actually measured in kilobits per second (or kbits/s), so 14.4 kbits/s, 28.8 kbits/s, and 33.6 kbits/s.
+
+A byte is considered a group of 8 bits, so 1 byte is equal to 8 bits. Historically, a byte was the number of bits used to encode a single character of text. Think about it this way: The binary code for the letter "a" is 01000001 and 01000001 is 8 bits. So each character is represented by 8 bits.
+
+One thing to keep in mind is bits are represented by a lower case "b" and a byte is represented by an upper case "B."
+
+## What Are Kilobytes?
+
+A kilobyte is approximately 1,000 bytes and since a byte is 8 bits, a kilobyte is roughly 8,000 bits. But why do we say "approximately" and "roughly"? A kilobyte is actually 1024 bytes.
```

Comment:
A kilobyte is 1000 bytes. A kibibyte is 1024 bytes. https://www.google.com/search?q=how+many+bytes+in+a+kilobyte&rlz=1C5CHFA_enUS998US998&oq=how+many+&aqs=chrome.0.69i59l3j69i57j69i59j69i61l2j69i60.1488j0j7&sourceid=chrome&ie=UTF-8